examples/user_managed_deps/BUILD.bazel

```diff
@@ -19,13 +18,11 @@ filegroup(
     visibility = ["//visibility:public"],
 )
 
-mock_typescript_lib(
-    name = "mock_ts_lib",
+filegroup(
```

Comment:
ts_library from source does not work yet. see https://github.com/bazelbuild/rules_nodejs/issues/1213